.github/workflows/python-package.yml

```
@@ -72,3 +72,7 @@ jobs:
     - name: Regression Tests
       run: |
         bash tests/runtests.sh
+    - name: Module tests without fusepy
+      run: |
+        pip uninstall -y fusepy
```

```suggestion
        python3 -m pip uninstall -y fusepy
```
I think this might be more stable because it does not require a `pip` binary installed in `PATH`. Having the module installed suffices. Also it would be more consistent with the line 38 in this file.

---

Done